### ISSUE TITLE
Easy clue emote steps QOL

### ISSUE THREAD

**Issue by Mylosas** [runelite/runelite#11786]
Suggestion to update indicator tiles for easy clue emote steps.
There are 31 emote step from easy clues.
-------------------------------------------------------------------------------------------------------
https://imgur.com/a/XuRhnh0
*Some of them don't need changing because they're a tile next to the stash.
**Some of them are only 1 tile further.
***Hard to decide which tile is the best for some steps because it can be reached from multiple directions depending on players progression level in-game. Mostly aimed at max efficiency.
---------------------------------------------------------------------------------------------------------
Imgur doesn't like links in description so I made pastebin for video proof: https://pastebin.com/N5UnaVgS

Yellow tiles with emote icon -> Current runelite indicators
Yellow tiles -> Closest tile from stash unit you can perform emote to proceed with the clue
Green tiles -> Used to visualize gaps

**Comment by makingcomment** [comment]
Why limit this to easy clues? All clue difficulties have emote clues.

**Comment by Broooklyn** [comment]
Probably because he's been grinding easy clues and noticed it

**Comment by Mylosas** [comment]
Correct, it's the same for all difficulties but I can't just post 'fix emote steps' without showing what the problem is, and for that you need to get emote step yourself, which would take quite some time for all clues. 
Someone mentioned that its different approach than #9601 but I'm assuming majority of the people doing easy clues use stash units, that's why it should be tied to it and not the middle point of area you can perform emote.
Also apparently 'emerald000' is working on ALL steps from beginner-hard clues so this thread might be just ignored/removed if it doesn't provide any help.

**Comment by Majst** [comment]
Been wanting this to get updated for so long, has anyone posted images for the other difficulty tiers?
